memote/utils.py

```diff
@@ -127,7 +127,8 @@ def get_ids(iterable):
 
 def get_ids_and_bounds(iterable):
     """Retrieve the identifier and bounds of a  number of objects."""
-    return [(elem.id, elem.lower_bound, elem.upper_bound) for elem in iterable]
+    return [elem.lower_bound + "< " + elem.id + "< " + elem.upper_bound for
```

Comment:
Sorry, since it should be `<=` anyhow, please use the format function, i.e.,
```python
return ["{0.lower_bound} <= {0.id} <= {0.upper_bound}".format(elem) for elem in iterable
```